robowflex_library/include/robowflex_library/planning.h

```diff
@@ -166,6 +167,72 @@ namespace robowflex
         std::condition_variable cv_;       ///< Planner condition variable
     };
 
+    /** \cond IGNORE */
+    ROBOWFLEX_CLASS_FORWARD(SimpleCartesianPlanner);
+    /** \endcond */
+
+    /** \class robowflex::SimpleCartesianPlannerPtr
+        \brief A shared pointer wrapper for robowflex::SimpleCartesianPlanner. */
+
+    /** \class robowflex::SimpleCartesianPlannerConstPtr
+        \brief A const shared pointer wrapper for robowflex::SimpleCartesianPlanner. */
+
+    /** \brief A simple motion planner that uses interpolation of the end-effector in Cartesian space to find
+     * a path. Valid configurations are found using IK. This planner is not complete and typically only works
+     * for small movements of the end-effector.
+     */
+    class SimpleCartesianPlanner : public Planner
+    {
+    public:
+        /** \brief Constructor.
+         */
+        SimpleCartesianPlanner(const RobotPtr &robot, const std::string &name = "");
+
+        // non-copyable
+        SimpleCartesianPlanner(SimpleCartesianPlanner const &) = delete;
+        void operator=(SimpleCartesianPlanner const &) = delete;
+
+        /** \brief Plan a Cartesian motion (interpolation) given a \a request and a \a scene from a \a start
```

Comment:
There's no scene in this method

Reply:
The scene is an optional field of the IKQuery here.